python/tests/test_create_graph.py

```diff
@@ -257,6 +258,7 @@ def test_loader_with_specified_data_type(
     ]
 
 
+@pytest.mark.skip(reason="waiting for vineyard support")
```

Comment:
Skip them inside the case when session is eager, and run them for lazy session.

Reply:
done